diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter6/classes/InlineClassesRule.kt

```diff
@@ -31,7 +31,11 @@ class InlineClassesRule(configRules: List<RulesConfig>) : DiktatRule(
     listOf(INLINE_CLASS_CAN_BE_USED)) {
     override fun logic(node: ASTNode) {
         val configuration = configRules.getCommonConfiguration()
-        if (node.elementType == CLASS && configuration.kotlinVersion >= ktVersion) {
+        if (node.elementType == CLASS &&
+                !(node.psi as KtClass).isInterface() &&
+                configuration.kotlinVersion >= minKtVersion &&
```

Comment:
In description you wrote, that we **shouldn't** suggest to use inline classes on **interfaces** and when version **greater than 1.4.x**. 

I see that you add check `!interface`, but why we call `handleClasses` when version > `1.3` and < `1.5.0`?
Shouldn't it be `configuration.kotlinVersion <= 1.4.0`?

Reply:
That's my mistake in the description; `inline` keyword is valid up to 1.5.0